File: Jack.Model/Model/KitItem.vb
<Serializable()> _
Public Class KitItem

#Region "Construtor"

    Public Sub New()
        Codigo = 0
        Kit = 0
        TipoItem = 0
        Observacao = String.Empty
    End Sub

#End Region

    Public Overridable Property Codigo As Integer
    Public Overridable Property Kit As Integer
    Public Overridable Property KitDescricao As String
    Public Overridable Property TipoItem As Integer
    Public Overridable Property Ordem As Integer
    Public Overridable Property TipoItemDescricao As String
    Public Overridable Property Observacao As String
    Public Overridable Property IsOpcional As String

    Public Overridable ReadOnly Property Opcional As String
        Get
            If IsOpcional = "S" Then
                Return "Opcional"
            Else
                Return "Não"
            End If
        End Get
    End Property

End Class

File: Jack.Model/Model/KitItem.cs
using System;
using Jack.Library.Extensions;
namespace Jack.Model
{
    [Serializable()]
    public class KitItem: BaseModel<KitItem>
    {

        #region "Construtor"

        public KitItem():base()
        {
            Codigo = 0;
            kit = new Kit();
            TipoItem = new TipoItem();
            Observacao = string.Empty;
        }

        #endregion

        private Kit kit;
        private TipoItem tipoItem;
        private int ordem;
        private string observacao;
        public virtual TipoItem TipoItem
        {
            get
            {
                return tipoItem;
            }
            set
            {
                tipoItem = value;
            }
        }
        public virtual int Ordem
        {
            get
            {
                return ordem;
            }
            set
            {
                ordem = value;
            }
        }

        public virtual string Observacao
        {
            get
            {
                return observacao;
            }
            set
            {
                observacao = value;
            }
        }

        public virtual Kit Kit
        {
            get
            {
                return kit;
            }
            set
            {
                kit = value;
            }
        }

    }
}
